otter/integration/lib/cloud_load_balancer.py

```diff
@@ -178,6 +178,41 @@ def delete(self, rcs, success_codes=None):
                 [202, 404] if success_codes is None else success_codes)
         ).addCallback(lambda _: rcs)
 
+    def list_nodes(self, rcs):
+        """
+        Get all the nodes on the load balancer.
+        But this function just returns the list of dictionaries.
```

Comment:
Oops!  Sorry, this was leftover from a previous edit.  Fixing - thanks for spotting that!